easybuild/easyconfigs/i/intltool/perl-5.26.0-compatibility.patch

```diff
@@ -0,0 +1,43 @@
+--- intltool-0.51.0.orig/intltool-update.in	2015-03-09 02:39:54.000000000 +0100
```

Comment:
@reedts Please include a description of why the patch is needed at the top of the patch. I know you've named it for compatibility, but a more specific description would be useful.

Reply:
Also: author or reference to upstream source of the patch.

We generally also include the software name/version in the filename.

If the Perl version is irrelevant (unclear to me), I'd avoid using `5.26.0` in the patch filename.

So, I'd suggest naming it `intltool-0.51.0-fix-Perl-regex.patch`...

Reply:
It seems to be needed for all versions of Perl since 5.23.0 
I will update this next week.